dojo/fixtures/test_type.json

```diff
@@ -726,5 +726,12 @@
         },
         "model": "dojo.test_type",
         "pk": 195
+    },
+    {
+        "fields": {
+            "name": "AuditJS Scan"
+        },
+        "model": "dojo.test_type",
+        "pk": 196
```

Comment:
This part is not needed. We load parser dynamically now.

Reply:
OK removed. It was automatically generated by CookieCutter